testing/yuca/ca/test_neural.py

```diff
@@ -197,6 +197,22 @@ def test_multiverse_to(self):
                     self.assertEqual(param[1].device.type, \
                             torch.device(my_device).type)
 
+    def test_kernel_params_config(self):
+
+        nca = NCA()
+
+        kparams = 1.0 * nca.kernel_params
+
+        ca_config = nca.make_config()
+
+        kconfig = ca_config["neighborhood_kernel_config"]
```

Comment:
This should be the same as the test in yuca/ca/test_continuous.py
The GaussianMixture neighborhood kerenl is more versatile, but it still might be useful to use the orbium style kernel (which takes `mu` and `sigma` as inputs instead of `parameters`

Reply:
fixed in fda8197